### PR TITLE
support /deep/ and >>>

### LINKED ISSUE #108: used <style scoped> BUG
### Expected behavior
@keyframes slidein {
  0%{
    opacity: 1;
  }
  100%{
    opacity: 0;
  }
}
### Actual behavior
@keyframes slidein {
  0%[data-v-04c2046b]{
    opacity: 1;
  }
  100%[data-v-04c2046b]{
    opacity: 0;
  }
}

### PR DESCRIPTION
**Pull request by sunnylost**
Follow vue-loader to support `/deep/` and `>>>`

also fixes #108 .

/ping @znck

### CODE REVIEW

**Comment by znck** [comment]
Cool. Thanks. Please add a covering test too.

**Comment by sunnylost** [comment]
ok, and i will fix the eslint warning.

File: src/style/css.js [hunk 1]
```diff
@@ -6,17 +6,56 @@ import camelcase from 'camelcase'
...
+ * @returns {boolean}
+ */
+function isInvalidTag (tag) {
+    let isValid = false
+
+    if (tag === 'from' ||
+        tag === 'to' ||
+        tag.match(/^\d/)
+    ) {
+        isValid = true
```
Comment (znck) [inline src/style/css.js]: You can directly return the expression in `if`.

Comment (znck) [inline src/style/css.js]: Regex .test would be better than tag.match

Comment (sunnylost) [inline src/style/css.js]: ok.

Comment (sunnylost) [inline src/style/css.js]: Can you check again?

**Pull request merged**